core/model-api/src/main/java/org/eclipse/rdf4j/model/Literal.java

```diff
@@ -61,6 +65,10 @@ default boolean isLiteral() {
 	 */
 	Optional<String> getLanguage();
 
+	default String getBaseDirection() {
```

Comment:
Can we make this method return `enum BaseDirection`?
You can call `getBaseDirection().getSuffix()` later.